Header header
int32 size
geometry_msgs/Point location
geometry_msgs/Point speed

float64[] location_covariance
float64[] speed_covariance

int32 SIZE_UNKNOWN=-1
int32 SIZE_SMALL=0
int32 SIZE_MEDIUM=1
int32 SIZE_LARGE=2

